dcat-implementation-report/index.html

```diff
@@ -28,19 +28,19 @@ <h3>Methodology</h3>
         <li>Identify the actual list of revisions proposed in [[VOCAB-DCAT-2]] for which implementation evidence should be collected (see <a href="#proposed-revisions"></a> for more details).</li>
         <li>Review existing vocabularies based on [[VOCAB-DCAT]], and identify if they include already the changes proposed in [[VOCAB-DCAT-2]], and/or their alignment with [[VOCAB-DCAT-2]] is planned. The rationale for including also planned implementations is that these vocabularies are not likely to adopt the revisions proposed in [[VOCAB-DCAT-2]] until it becomes a W3C Recommendation.</li>
         <li>Review existing guidelines and implementation reports of [[VOCAB-DCAT]], as well as of vocabularies based on it, to verify if the changes proposed [[VOCAB-DCAT-2]] are implemented in existing data catalogs, data services, and datasets.</li>
-        <li>The Implementation Report was developed.</li>
+        <li>Develop an Implementation Report with details of the steps above.</li>
         </ol>
 
         <p>As noted, to have a broader coverage of [[VOCAB-DCAT-2]] adoption we considered different types of evidence:</p>
         <ol>
           <li>DCAT-based vocabularies</li>
           <li>Data catalogs, data services, and datasets</li>
         </ol>
-        <p>Because of the number of DCAT-based vocabularies and data catalogs supporting [[VOCAB-DCAT]], this report includes only a representative subset, providing nonetheless enough implementation evidence of the revisions proposed in [[VOCAB-DCAT-2]].</p>
+        <p>Due to the large number of DCAT-based vocabularies and data catalogs supporting [[VOCAB-DCAT]], this report includes only a representative subset, providing nonetheless enough implementation evidence of the revisions proposed in [[VOCAB-DCAT-2]].</p>
       </section>
       <section id="exit-criteria">
         <h3>Meeting the exit criteria</h3>
-        <p>As described in the <abbr title="Dataset Exchange Working Group">DXWG</abbr> <a href="https://www.w3.org/2017/dxwg/charter">charter</a>, to move on to Proposed Recommendation, evidence will be adduced in order to demonstrate that each of the changes made to the original DCAT Recommendation [[VOCAB-DCAT]] has been published and consumed independently in catalogs and related systems at least once, although a higher number is expected for the majority of terms.</p>
+        <p>As described in the <a href="https://www.w3.org/2017/dxwg/charter"></a><abbr title="Dataset Exchange Working Group">DXWG</abbr> charter</a>, to advance to Proposed Recommendation, evidence will be adduced in order to demonstrate that each of the changes made to the original DCAT Recommendation [[VOCAB-DCAT]] has been published and consumed independently in catalogs and related systems at least once, although a higher number is expected for the majority of terms.</p>
```

Comment:
I would keep the link to the charter, as allows readers to directly check the success criteria for DCAT, which are different from the usual ones of having 2 independent implementations.

Reply:
my intention was to keep the link, only to expand it to the phrase 'DXWG charter', will revise

Reply:
@andrea-perego now back to the original link to 'charter'